pallene/print_ir.lua

```diff
@@ -217,7 +217,7 @@ local function Cmd(cmd)
     elseif tag == "ir.Cmd.NewRecord"  then rhs = "new ".. cmd.rec_typ.name .."()"
     elseif tag == "ir.Cmd.GetField"   then rhs = Field(cmd.src_rec, cmd.field_name)
     elseif tag == "ir.Cmd.SetField"   then rhs = Val(cmd.src_v)
-    elseif tag == "ir.Cmd.CallStatic" then rhs = Call(Fun(cmd.f_id),  Vals(cmd.srcs))
+    elseif tag == "ir.Cmd.CallStatic" then rhs = Call(Fun(cmd.src_f.id),  Vals(cmd.srcs))
```

Comment:
I get nervous when I see two field accesses in a row, without checking the type tag... In this case we might be able to avoid this if we use `Val(cmd.src_f)` instead of `Fun(cmd.src_f.id)`.